Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml

```diff
@@ -43,6 +43,9 @@
   - type: Food
     trash: 
     - FoodPacketBoritosTrash
+  - type: Tag
+    tags:
+      - VoxFood
```

Comment:
Please fix all the VoxFood indentations, the `-` should be aligned with its parent